examples/step-57/step-57.cc

```diff
@@ -172,31 +168,20 @@ namespace Step57
     return 0;
   }
 
-  template <int dim>
-  void BoundaryValues<dim>::vector_value(const Point<dim> &p,
```

Comment:
This is a standard thing we have to avoid indirect function calls. I am okay to remove it, though.

Reply:
Yes. I find these distracting in tutorial programs since they are usually not relevant to the example itself.